groovy-mode.el

```diff
@@ -749,19 +749,43 @@ Then this function returns (\"def\" \"if\" \"switch\")."
         (setq syntax (syntax-ppss (point)))))
     paren-depth))
 
+(defun groovy--extract-line-without-comments ()
+  "Extracts the part of the current line that is not a comment."
+  (let (code-text
+		(start-pos (line-beginning-position))
```

Comment:
Would you mind indenting with spaces instead? That's what the rest of the file uses.